frontend/wasm/src/api.rs

```diff
@@ -306,6 +306,11 @@ impl Editor {
 		self.dispatch(message);
 	}
 
+	pub fn select_layer(&self, paths: Vec<LayerId>, ctrl: bool, shift: bool) {
```

Comment:
this does not look ffi safe, I would expect there to be an issue with bigints not being sendable across the js/rs boundary

Reply:
This was how this was being done previous as far as I am aware. I just checked prior to my commit and similar examples seem to exist. Recommendations on how to proceed?

Reply:
Actually, nevermind, I think this could work: https://rustwasm.github.io/wasm-bindgen/reference/types/number-slices.html
(If not, we would have to turn paths into &[LayerId])
This just doesn't work for nested arrays

Reply:
The wasm-bindgen docs don't explicitly mention Vec as a supported type (or at least I didn't see it) but that might just be a lack of documentation

Reply:
You can leave it as it its I just got confused by the documentation